### PR TITLE
Add basic mail functionality to the contact form

### PR DESCRIPTION
**Pull request by mnk87**
this branch implements a basic mailing functionality to the contact form, without the queue functionality built in, this is set back to maybe fix on some other time but for now this works.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -48,3 +48,13 @@ MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
...
+
+# below is the settings you need to use with your own username and password to make mailing work
+# this comes instead of the other mail block in the env file
+MAIL_DRIVER=smtp
+MAIL_HOST=smtp.mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_FROM_ADDRESS=from@example.com
+MAIL_FROM_NAME=molveno
```
Comment (mtheunissen82) [inline .env.example]: De QUEUE_DRIVER variabele ook nog toevoegen

**Review by mtheunissen82 — DISMISSED** [review]

File: .env.example (hunk 1 above)
Comment (mnk87) [reply .env.example]: na overleg hebben we besloten dat dit niet hoeft

File: app/Http/Controllers/ContactFormController.php [hunk 2]
```diff
@@ -43,6 +45,7 @@ public function send(Request $request)
         $contact_form->message = $request->input('message');
 
         if ($contact_form->save()) {
+            Mail::to('admin@molveno.it')->send(new ContactFormEmail($contact_form));
```
Comment (HonkingGoose) [inline app/Http/Controllers/ContactFormController.php:48]: PHPStorm complains: "Undefined class Mail". I'm not sure if this is actually a problem or not...

File: app/Mail/ContactFormEmail.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this->view('mail.contact')
+                    ->with(['contactForm' => $this->contactForm]);
```
Comment (HonkingGoose) [inline app/Mail/ContactFormEmail.php:35]: PHPStorm complains: Property `contactForm` not found in ContactFormEmail

File: resources/views/mail/contact.blade.php [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h2>{{ $contactForm->firstName }} {{ $contactForm->lastName }} has said the following:</h2>
```
Comment (HonkingGoose) [inline resources/views/mail/contact.blade.php:1]: Please check if the headers are correct now, I've changed them from a h1 and h2 into a h2 and h3.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (HonkingGoose) [reply .env.example]: Hint: You can close a conversation as resolved by selecting "Resolve conversation" in the GitHub interface.

File: app/Http/Controllers/ContactFormController.php (hunk 2 above)
Comment (mnk87) [reply app/Http/Controllers/ContactFormController.php:48]: it's not, i imported the class mail at the top of this file so he should recognize it. When I tested it, it actually sent an email to my mailtrap inbox so it works as well

File: resources/views/mail/contact.blade.php (hunk 4 above)
Comment (mnk87) [reply resources/views/mail/contact.blade.php:1]: fine by me

File: app/Mail/ContactFormEmail.php (hunk 3 above)
Comment (mnk87) [reply app/Mail/ContactFormEmail.php:35]: that's actually something I forgot to declare as a property for the class but it still works.. I'll add it anyway in a new commit in a sec.

Comment (mnk87) [reply app/Mail/ContactFormEmail.php:35]: error should be gone now

**Review by HonkingGoose — APPROVED** [review]

**Pull request merged**